src/openllm/models/baichuan/configuration_baichuan.py

```diff
@@ -36,6 +36,8 @@ class BaichuanConfig(openllm.LLMConfig):
         "default_id": "baichuan-inc/baichuan-7b",
         "model_ids": [
             "baichuan-inc/baichuan-7b",
+            "baichuan-inc/baichuan-13b-base",
```

Comment:
Hey @aarnphm, it is cool to add baichuan-13b models. However, I noticed that `cpm-kernels` is required by the `baichuan-13b` models, do we need to add `cpm-kernels` as the requirements for baichuan model? Any thoughts?

Reference: https://github.com/baichuan-inc/Baichuan-13B/blob/main/requirements.txt#L3

Reply:
oh i see, let me fix that. I can add it to requirements

Reply:
I've created a PR https://github.com/bentoml/OpenLLM/pull/117 to fix the dependency issue, it would be genuinely appreciate if u could kindly review it.